Updating Managing Clusters Main Page (#3014)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1.0-beta/clusters/index.md

Kommander allows you to monitor and manage very large numbers of clusters and so we make it easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.

## Before you begin:

> You will already need to have run `konvoy up` with the Kommander addon deployed. You should be looking at the Kommander UI.

Reviewer: ```suggestion
You need to have run `konvoy up` with the Kommander addon deployed. You should be looking at the Kommander UI.
```

> If you are creating a cluster with a cloud partner (Azure, AWS), you will need to configure your [infrastructure provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/).

Reviewer: ```suggestion
If you are creating a cluster with a cloud partner (Azure, AWS), you need to configure your [infrastructure provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/).
```

> If you are planning on attaching more than one cluster, make sure to enter in your valid [license](/ksphere/kommander/1.1.0-beta/licensing/).

Reviewer: ```suggestion
If you are planning on attaching more than one cluster, ensure you enter your valid [license](/ksphere/kommander/1.1.0-beta/licensing/).
```

From the dashboard page or the clusters page, click Add Cluster and Create Konvoy Cluster.

> ![Add Cluster Options](/ksphere/kommander/1.1.0-beta/img/add-cluster-beta-1-1-0.png)

Reviewer 2: 'nit: we dont need to have version in filename since the whole img folder already is in its versions folder 👍

Author: I realize this - I just named it different so I could keep track when I was making screenshots. I'm indifferent, seems more of an effort to change back now 🤷

> ### Creating a Konvoy Cluster

Reviewer 2: hm, Kommander is capable of creating clusters on its on, but these links only point to konvoy?
(might be unrelated to this PR tho)

Author: yeah, unrelated.

- [Creating Konvoy Clusters on AWS](/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-aws/)
- [Creating Konvoy Clusters on Azure](/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-azure/)
- [Creating Konvoy Clusters on your own infrastructure](/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-on-prem/)

### Connecting an Existing Cluster

Using the Add Cluster option, you can attach an already existing cluster, even an existing Konvoy cluster, directly to Kommander. Enjoy all the multi-cluster management and monitoring benefits that Kommander provides while keeping your existing cluster on its current provider & infrastructure.

See [Attach Cluster](/ksphere/kommander/1.1.0-beta/clusters/attach-cluster/)

### Creating a Konvoy Cluster via YAML

> Clicking the Upload YAML option will allow you to utilize a config generated by the Konvoy CLI tool. An example config can be created in the file `cluster.yaml` by [running](/ksphere/konvoy/1.5.0-beta/reference/command-line-interface/konvoy-init/) `konvoy init`, or one is generated after running `konvoy up`. Ensure the correct Cloud Provider credentials are selected and add any cluster labels that apply.

Reviewer: ```suggestion
Selecting the **Upload YAML option** allows you to use a config generated by the Konvoy CLI tool. An example config can be created in the file `cluster.yaml` by [running](/ksphere/konvoy/1.5.0-beta/reference/command-line-interface/konvoy-init/) `konvoy init`, or one is generated after running `konvoy up`. Ensure the correct Cloud Provider credentials are selected and add any cluster labels that apply.
```

![Create Cluster Upload YAML](/ksphere/kommander/1.1.0-beta/img/create-cluster-yaml.png)

#### Adding Infrastructure Provider Tags

> A tag is similar to a label and can help manage and track this cluster at the infrastructure provider level along with other provider resources. As providers support different amounts of tags, up to 10 tags may be added here as baseline. You can always add more directly via your infrastructure provider.

Reviewer: ```suggestion
A tag is similar to a label and helps manage and track this cluster at the infrastructure provider level along with other provider resources. As providers support different amounts of tags, up to 10 tags may be added here as a baseline. You can add more directly through your infrastructure provider.
```

#### Adding/Removing Cluster Labels

> Cluster labels are matched to the selectors created for [projects](/ksphere/kommander/1.1.0-beta/projects/). If a cluster is removed from a project any resources deployed to the cluster from that Project will be removed. If a cluster is added to a project any existing project resources will be deployed to the cluster.

Reviewer: ```suggestion
Cluster labels are matched to the selectors created for [projects](/ksphere/kommander/1.1.0-beta/projects/). If a cluster is removed from a project, any resources deployed to the cluster from that Project are removed. If a cluster is added to a project any existing project resources are deployed to the cluster.
```

#### Valid labels

> - Valid labels must have a key and a value, separated by a colon and a space. For example, "env: dev" (if creating through the UI, the space is automatically created from entering in your values in the textboxes)

Reviewer: ```suggestion
- Valid labels must have a key and a value, separated by a colon and a space. For example, "env: dev". If you are creating the label through the UI, the space is automatically added when entering your values in the fields.
```
